File: 448. Find All Numbers Disappeared in an Array/Solution.cs
public class Solution {
    public IList<int> FindDisappearedNumbers(int[] nums) {

        foreach (int num in nums) {
            int positiveNum = Math.Abs(num);
            if (nums[positiveNum-1] > 0) nums[positiveNum-1] *= -1;
        }

        var disappearedNums = new List<int>();

        for (int i = 0; i < nums.Length; ++i) {
            if (nums[i] > 0) disappearedNums.Add(i+1);
        }
        return disappearedNums;
    }
}

File: 448. Find All Numbers Disappeared in an Array/solution.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:

        for num in nums:
            positiveNum = abs(num)
            if nums[positiveNum-1] > 0:
                nums[positiveNum-1] *= -1
        

        disappearedNums = []

        for i in range(len(nums)):
            if nums[i] > 0:
                disappearedNums.append(i+1)
        
        return disappearedNums
